test/AccordionSpec.tsx

```diff
@@ -9,183 +10,181 @@ import Nav from '../src/Nav';
 
 describe('<Accordion>', () => {
   it('should output a div', () => {
-    mount(<Accordion />).assertSingle('div');
+    const { getByTestId } = render(<Accordion data-testid="test" />);
+
+    getByTestId('test').tagName.toLowerCase().should.equal('div');
   });
 
   it('should render flush prop', () => {
-    mount(<Accordion flush />).assertSingle('.accordion.accordion-flush');
+    const { getByTestId } = render(<Accordion flush data-testid="test" />);
+
+    const node = getByTestId('test');
+    node.classList.contains('accordion').should.be.true;
+    node.classList.contains('accordion-flush').should.be.true;
   });
 
   it('should output a h1', () => {
-    const wrapper = mount(
+    const { getByTestId } = render(
       <Accordion>
         <Accordion.Button>Hi</Accordion.Button>
-        <AccordionCollapse as="h1" eventKey="0">
+        <AccordionCollapse as="h1" eventKey="0" data-testid="test-collapse">
           <span>hidden Data</span>
         </AccordionCollapse>
       </Accordion>,
     );
 
-    wrapper.find('AccordionCollapse').assertSingle('h1');
+    getByTestId('test-collapse').tagName.toLowerCase().should.equal('h1');
   });
 
   it('should only have second item collapsed', () => {
-    const wrapper = mount(
+    const { getByTestId } = render(
       <Accordion defaultActiveKey="0">
-        <Accordion.Item eventKey="0">
+        <Accordion.Item eventKey="0" data-testid="item-0">
           <Accordion.Header />
           <Accordion.Body>body text</Accordion.Body>
         </Accordion.Item>
-        <Accordion.Item eventKey="1">
+        <Accordion.Item eventKey="1" data-testid="item-1">
           <Accordion.Header />
           <Accordion.Body>body text</Accordion.Body>
         </Accordion.Item>
       </Accordion>,
     );
-    const collapses = wrapper.find('AccordionCollapse');
 
-    collapses.at(0).getDOMNode().className.should.include('show');
-    collapses.at(1).getDOMNode().className.should.include('collapse');
+    expect(getByTestId('item-0').querySelector('.show')).to.exist;
+    expect(getByTestId('item-1').querySelector('.collapse')).to.exist;
```

Comment:
It would look a little bit cleaner if we stick to convention and check `classList`. It would also better mimic the behavior of previous test. Something similar to this
```
    getByTestId('item-0').classList.contains('show').should.be.true;
    getByTestId('item-1').classList.contains('collapse').should.be.true;
```
might differ a bit from this of course

Reply:
`show` and `collapse` classes are set on `accordion-collapse` element (`AccordationCollapse` component) which is not the were data-testid is set. So that's why I'm using `querySelector`.

----
Snippet from test:
<img width="375" alt="image" src="https://user-images.githubusercontent.com/24919330/152199529-ca444bec-9319-4b71-8f4e-68d8f2d1d077.png">

To add more context I can add `.accordion-collapse` class to the selector

Reply:
oh, I didn't realize that, all is good then 👍